tests/typescript_index_signature/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -11,11 +11,14 @@ class LocalStorage implements Storage {
   [key: string]: any;
 }
 
+type A = { [key: string] };
```

Comment:
As an index signature without a type annotation is not valid TS, please move the test for this case to a separate file in `tests/typescript_error_recovery`.